packages/x-date-pickers/src/locales/utils/pickersLocaleTextApi.ts

```diff
@@ -61,7 +61,13 @@ export interface PickersComponentAgnosticLocaleText<TDate extends PickerValidDat
   todayButtonLabel: string;
 
   // Clock labels
-  clockLabelText: (view: TimeView, time: TDate | null, adapter: MuiPickersAdapter<TDate>) => string;
+  clockLabelText: (
+    view: TimeView,
+    time: TDate | null,
+    utils: MuiPickersAdapter<TDate>,
```

Comment:
Nitpick: should we maybe deprecate these as well? 🤔

Reply:
Done
Not sure this deprecation warning is actually displayed anywhere but it will help us for v8 :laughing: